#                                   HYPOTHESIS TESTING

### Solution

In [13]:
import pandas as pd
import numpy as np
import scipy.stats as stats
import math

#### Step1: Hypothesis Statement

#### Step2:Calculate Test Statistic

In [14]:
x_bar = 3050     
population_mean = 1000 + 5*600 
sigma = 5*25      # standard deviation
n = 25            # sample size

In [15]:
t = (x_bar - population_mean) / (sigma / math.sqrt(n))
t

-38.0

#### Step3:Determine Probability and comparing

In [16]:
z_critical = stats.norm.ppf(0.95)
print(f'Z_critical value is {z_critical:.3f}')

# Decision
if t > z_critical:
    decision = "Reject H0"
else:
    decision = "Fail to Reject H0"

t, z_critical, decision


Z_critical value is 1.645


(-38.0, np.float64(1.6448536269514722), 'Fail to Reject H0')

#### Step4:Make a Decision: 
Z=-38,Zcritical=1.645.Since Z<Zcritical the test statitics doesnot fall in rejection region.FAIL TO REJECT THE NULL HYPOTHESIS(H0).

#### Step5: Conclusion:

In [17]:
import pandas as pd

data = {
    "Parameter": ["Sample Mean (x̄)", "Theoretical Mean (μ)", "Standard Deviation (σ)", 
                  "Sample Size (n)", "Test Statistic (t)", "Critical Value (Z₀.₀₅)", "Decision"],
    "Value": [x_bar, population_mean, sigma, n, round(t, 2), z_critical, decision]
}

summary_df = pd.DataFrame(data)
summary_df

Parameter              Value
0        Sample Mean (x̄)               3050
1    Theoretical Mean (μ)               4000
2  Standard Deviation (σ)                125
3         Sample Size (n)                 25
4      Test Statistic (t)              -38.0
5  Critical Value (Z₀.₀₅)           1.644854
6                Decision  Fail to Reject H0

## Conclusion

### Z-score (-38.00) is smaller than the critical value (1.64),
### So, we fail to reject the null hypothesis (H₀),This means there is no strong proof that the weekly operating costs are higher than what the model shows.